src/app/problem/right/Block.js

```diff
@@ -0,0 +1,24 @@
+
+
+const Block = ({id, title, titleButton, children, style}) => {
+    return <>
+        <div id={id} style={{width: "100%", ...style}}>
+            <div className="block-title" style={{
```

Comment:
You can write JSDoc to help other contributors easy to know the meaning of parameters.
Here is an example:
![image](https://user-images.githubusercontent.com/41495987/131280015-c20457e2-91ad-41b0-aa56-6b2c8e4908f4.png)

Reply:
This is a great idea, I will add it!